features/prodigy.feature

```diff
@@ -33,3 +33,12 @@ Feature: Prodigy
       | baz  | nil         |
       | foo  | nil         |
       | qux  | nil         |
+
+  Scenario: Prodigy mode
```

Comment:
we should be a little more explicit here. What is the scenario that this test is checking?